portal-test/src/com/liferay/portal/test/property/PortalPropertyTestUtil.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.test.property;
+
+import com.liferay.petra.reflect.ReflectionUtil;
+import com.liferay.portal.util.PropsValues;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+
+/**
+ * @author Alicia García
+ */
+public class PortalPropertyTestUtil {
```

Comment:
Why not call it `PropsValuesTestUtil`? And maybe rename the method to `setProperty`?

Reply:
There is another `PropsValuesTestUtil` on` src/com/liferay/portal/kernel/test/util/PropsValuesTestUtil.java`

I do not think a good idea call it the same way.

Reply:
Mmm, cannot we use the logic that's already there? It seems to be doing (almost) the same this.